html2docx/image.py

```diff
@@ -0,0 +1,91 @@
+import io
+import pathlib
+import time
+import urllib.error
+import urllib.request
+from typing import Dict, Optional
+
+from docx.image.exceptions import UnrecognizedImageError
+from docx.image.image import Image
+from docx.shared import Inches
+
+USABLE_HEIGHT = Inches(8.1)
+USABLE_WIDTH = Inches(5.8)
+
+MAX_IMAGE_SIZE = 10 * 1024 * 1024  # 10 MiB
+
+
+def load_image(src: str) -> io.BytesIO:
+    image_buffer = None
+    retry = 3
+    while retry and not image_buffer:
+        try:
+            with urllib.request.urlopen(src) as response:
+                size = response.getheader("Content-Length")
+                if size and int(size) > MAX_IMAGE_SIZE:
+                    break
+                data = response.read(MAX_IMAGE_SIZE + 1)
```

Comment:
Sorry if the answer is obvious, but what is the +1 for?

Could potentially use a comment.

Reply:
When `MAX_IMAGE_SIZE` bytes are read, either
- the last byte read was the end of the image, then `len(data) == MAX_IMAGE_SIZE` and this is the end of the file (otherwise `MAX_IMAGE_SIZE + 1` bytes would have been read),
- or the image is at least `MAX_IMAGE_SIZE + 1` bytes and considered too big.

I can add a comment.

Reply:
Ah, I see. This is for when the `Content-Length` header does not exist. Is that right?

Reply:
Yes, exactly.

Reply:
Thanks. I still think this could use a short comment that explains this is used when the header doesn't exist.